templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -28,7 +28,7 @@ import * as gax from 'google-gax';
 import {Callback, CallOptions, Descriptors, ClientOptions
 {%- if service.longRunning.length > 0 %}, LROperation{%- endif -%}
 {%- if service.paging.length > 0 %}, PaginationCallback, GaxCall{%- endif -%}
-{%- if service.iamService > 0 %}, IamClient, IamProtos{%- endif -%}
+{%- if (service.IAMPolicyMixin) > 0 %}, IamClient, IamProtos{%- endif -%}
```

Comment:
Brackets are not needed (here and below).

Reply:
Thank you! Fixed.